src/tasks/AotCompilerTask/MonoAOTCompiler.cs

```diff
@@ -106,6 +106,16 @@ public class MonoAOTCompiler : Microsoft.Build.Utilities.Task
     /// </summary>
     public bool UseDirectPInvoke { get; set; }
 
+    /// <summary>
+    /// PInvokes to call directly.
+    /// </summary>
+    public string[] DirectPInvokes { get; set; } = Array.Empty<string>();
```

Comment:
Since the use from msbuild would be an item group, maybe the msbuild parameter should be of type ITaskItem instead since it gives you more flexibility going forward with protentional additional metadata + calling the task from msbuild will be simpler, just passing @(DirectPInvokes) as the parameter.

Reply:
From msbuild project side you can still pass in a `@(DirectPinvokes)`. It will get mapped to the `ItemSpec`/`Include` value being the strings here.

Reply:
OK, but I still don't see why you would use a string array instead of a ITaskItem array in this case and keeping it as an ITaskItem array also opens up the ability to pass along metadata in the future, if ever needed.

Reply:
There is a slight convenience in using string.Join if `DirectPInvokes` were an array of strings instead of having to extract the Identity metadata for each item in the ITaskItem.

I modified it to be an ITaskItem and am using a string builder instead.